pkg/cluster/install.go

```diff
@@ -29,6 +29,8 @@ import (
 func (m *manager) AdminUpdate(ctx context.Context) error {
 	steps := []steps.Step{
 		steps.Action(m.initializeKubernetesClients), // must be first
+		// TODO: We should not be relaying on CSP availability in admin actions. Remove this once we fixup this on all clusters.
```

Comment:
Im bit tempted to not to add this before cluster credential rotation is in place. If credentials are expired this will block adminUpdates... Or we should add this to the bottom and leave for a while.